### DISCUSSION TITLE
Any way to load from a file?

### DISCUSSION THREAD

**Discussion by Imzachjohnson** [alirezamika/autoscraper#80] [Q&A] [answered]
Any native way to load in an html file and process it instead of a url? I am using a headless browser to load a javascript heavy website and therefore can't load the URL directly.

**Comment by alirezamika** [comment, marked as answer]
You can use the `html` parameter to pass the html content instead of the `url` parameter.

**Reply by Imzachjohnson** [reply]
Thank you. Found it right after I asked of course :/

**Reply by Imzachjohnson** [reply]
amazing tool by the way! making my life so much easier. Thank you.
